### PR TITLE
[candi] Remove k8s 1.19 support

### PR DESCRIPTION
**Pull request by yalosev**
## Description
Remove support of the 1.19 k8s version

## Why do we need it, and what problem does it solve?
It's outdated. We don't want to handle in anymore

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: candi
type: chore
summary: Remove Kubernetes version 1.19 support
impact_level: high
impact: |
  Migrate to the Kubernetes version v1.20+ to upgrade to the current Deckhouse release.
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @yalosev. I've started the workflow for label 'e2e/run/aws'...

File: docs/site/pages/index.html [hunk 1]
```diff
@@ -62,7 +62,7 @@ <h1 class="product-intro__title text_lead">
           </svg>
         </div>
         <div class="product-intro__certificate--text">
-          <span>Certified in CNCF</span>for Kubernetes 1.19—1.23
+          <span>Certified in CNCF</span>for Kubernetes 1.20—1.23
```
Comment (z9r5) [inline docs/site/pages/index.html:65]: @nabokihms Would it be better to leave it as it is?

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by z9r5** [comment]
@yalosev What about Ubuntu 16.04 as a supported OS for nodes?

**Review by kvaps — APPROVED** [review]

File: docs/site/pages/index.html (hunk 1 above)
Comment (yalosev) [reply docs/site/pages/index.html]: rollbacked

**Comment by yalosev** [comment]
Hm, OS is not directly connected with k8s version

**Comment by z9r5** [comment]
> Hm, OS is not directly connected with k8s version

It was just a question. Ok. I thought maybe it's time to delete 16.04 :)

File: candi/version_map.yml [hunk 2]
```diff
@@ -187,32 +187,6 @@ k8s:
...
-      # etcd: sha256 digest isn't needed because this component is compiled from source
-      # kubeApiserver: sha256 digest isn't needed because this component is compiled from source
-      # kubeControllerManager: sha256 digest isn't needed because this component is compiled from source
-      kubeScheduler: sha256:1a335251eaef4e209d5757da0bf5499ecdce6e65413f7cb92ff4cc633d6fc7dd
-      kubeProxy: sha256:092f9526686d27964d17be772c42cde086690209cc8aea10c49991456eb879c2
-  '1.20':
-    status: available
-    patch: 15
-    cniVersion: 0.8.7
-    bashible: *bashible_k8s_ge_1_19
```
Comment (RomanenkoDenys) [inline candi/version_map.yml:215]: Why we remove this code ?

**Review by RomanenkoDenys — COMMENTED** [review]
one small question

File: candi/version_map.yml (hunk 2 above)
Comment (yalosev) [reply candi/version_map.yml:215]: I have changed `bashible: &bashible_k8s_ge_1_19` to `bashible: &bashible_k8s_ge_1_20`. All other staff - it's a git magic. Please look at line 159 here

**Pull request merged**